### PR TITLE
New package: kevinboss.port version 1.0.9

### PR DESCRIPTION
**Pull request by kevinboss**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/88115)

### CODE REVIEW

**Comment by kevinboss** [comment]
@microsoft-github-policy-service agree [company="Kevin Boss"]

**Comment by kevinboss** [comment]
@microsoft-github-policy-service agree

File: manifests/k/kevinboss/port/1.0.9/kevinboss.port.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+PackageVersion: 1.0.9
+PackageLocale: en-US
+Publisher: Kevin Boss
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: port
+# PackageUrl:
+License: GPL
```
Comment (Trenly) [inline manifests/k/kevinboss/port/1.0.9/kevinboss.port.locale.en-US.yaml:14]:
```suggestion
License: GPL-3.0
```

File: manifests/k/kevinboss/port/1.0.9/kevinboss.port.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# LicenseUrl:
+# Copyright:
+# CopyrightUrl:
+ShortDescription: a tool that has been designed to manage multiple docker images and / or tags of these images. It allows the user to run one of these images / tags in a container, creating snapshots of that running container and manage the downloaded images
+# Description:
```
Comment (Trenly) [inline manifests/k/kevinboss/port/1.0.9/kevinboss.port.locale.en-US.yaml:19]:
```suggestion
ShortDescription: A tool that has been designed to manage multiple docker images and / or tags of these images.
Description: |-
  A tool that has been designed to manage multiple docker images and / or tags of these images. 
  It allows the user to run one of these images / tags in a container, creating snapshots of that running container and manage the downloaded images
```

**Review by Trenly — CHANGES_REQUESTED** [review]
@kevinboss - Are you aware that `port -v` shows an incorrect version?
image

**Comment by kevinboss** [comment]
> @kevinboss - Are you aware that `port -v` shows an incorrect version?
> 
> image
> 
> 

I am, that was never correct so far but not something with high priority.
I don't want to modify files during the build process nor do I want to manually alter the version number. So until I figure out how to set the version number during build this will be wrong.

**Review by Trenly — APPROVED** [review]

**Pull request merged**